tracer/test/test-applications/integrations/Samples.HttpMessageHandler/RequestHelpers.cs

```diff
@@ -173,7 +173,7 @@ void Invoke()
             SynchronizationContext.SetSynchronizationContext(null);
         }
 
-#if NET5_0
+#if NET5_0 || NET6_0
```

Comment:
```suggestion
#if NET5_0_OR_GREATER
```

Reply:
Done in https://github.com/DataDog/dd-trace-dotnet/pull/1885/commits/242c3c813e05991fdf1543aa6f7574ce577c6805